pkg/services/sqlstore/migrator/migrator.go

```diff
@@ -5,21 +5,29 @@ import (
 	"time"
 
 	_ "github.com/go-sql-driver/mysql"
+	"github.com/golang-migrate/migrate/v4/database"
```

Comment:
I have added it as dependency since I use some utility functions. Alternative I could have implemented inside grafana. Not sure which we prefer better.